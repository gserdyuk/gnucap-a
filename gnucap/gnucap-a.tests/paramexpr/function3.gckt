* functions use other functions

.param f1()= 'exp(1)'
.param f2(x)  = 'x+sin(x)'
.param f3(x,y)= 'exp(x)+f2(y)'

* f1 = 2.718281828 
v1 1 0 dc 'f1()'
* f2(1)=1+sin(1)=1.8414709
v2 2 0 dc 'f2(1)'
* f3 =2.718281828+1.8414709 = 4.55975
v3 3 0 dc 'f3(1,1)'

.print op v(v*) 

.op

.end
