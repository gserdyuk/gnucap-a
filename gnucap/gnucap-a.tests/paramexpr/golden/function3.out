Gnucap gnucap-arails  2009.12.07 RCS 26.136
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
* functions use other functions 
#           v(v1)      v(v2)      v(v3)     
 27.        2.7183     1.8415     4.5598    
